services-core/src/orderbook/streamed/updating_orderbook.rs

```diff
@@ -73,6 +73,11 @@ impl UpdatingOrderbook {
                             "Failed to construct orderbook from path (using default): {:?}",
                             error,
                         );
+                    } else {
+                        warn!(
+                            "Orderbook at specified path not found: {:?}",
+                            error,
+                        );
```

Comment:
Looks like this unrelated change snuck in? It's so small that I don't mind including it here, just mentioning.

Reply:
Yes, seemingly unrelated, but it all relates back to be fixing some stuff for the tutorial and I realized that when I don't put the right path, it doesn't tell me anything about what happened to my orderbook file config. Thanks for allowing it!

Reply:
I also think its fine including it. Especially since, as I imagine, it was added while debugging for the current PR.